packages/jest-util/src/index.js

```diff
@@ -84,7 +87,14 @@ const warnAboutUnrecognizedOptions  = (argv: Object, options: Object) => {
     !allowedOptions.has(arg)
   ));
   if (unrecognizedOptions.length) {
-    console.warn('Unrecognized options: ' + unrecognizedOptions.join(', '));
+    const title = `${BULLET} Unrecognized CLI Options Error`;
+    const message =
+      `  These options were not recognized by Jest:\n` +
+      `  ${chalk.bold(format(unrecognizedOptions))}`;
+    const comment =
+      `  ${chalk.bold('CLI Options Documentation')}:\n` +
+      `  http://facebook.github.io/jest/docs/api.html#jest-cli-options\n`;
```

Comment:
This will be on `/jest/docs/cli.html` after we merge new-docs (next week?). Mind using that already? We won't be tagging a release until then.

See https://jest-bot.github.io/jest/docs/cli.html